Packager/App.config

```diff
@@ -52,7 +52,9 @@
     
     <!-- web-service-->
     <add key="PodAuthorizationFile" value="c:\dependencies\pod-Auth.xml"/>
-    <add key="WebServiceUrl" value="https://pod.mdpi.iu.edu/"/>
```

Comment:
Need to revert to PROD POD

Reply:
resolved in https://github.com/IUMDPI/IUMediaHelperApps/pull/29/commits/081fa40ab3ff9a7b4c4829c0db701319db770d8b